product-service/product-service/src/lambda-functions/create-product.ts

```diff
@@ -0,0 +1,39 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import 'source-map-support/register';
+import productService from '../services/product.service';
+import { headers } from '../constants/headers';
+
+export const createProduct: APIGatewayProxyHandler = async (event) => {
+    console.log(`httpMethod: ${event.httpMethod}; path: ${event.path}; body: ${event.body};`);
+
+    try {
+        const body = event.body;
+
+        const createdProduct = await productService.createProduct(JSON.parse(body));
+
+        if (createdProduct) {
+            return {
+                headers,
+                statusCode: 200,
+                body: JSON.stringify(createdProduct)
+            };
+        } else {
+            return {
```

Comment:
You can also throw from here so the same 500 error will be returned from the catch :)